packages/dev-tools-pages/ts/index.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+import { MetaTags } from 'ts/components/meta_tags';
+import { Landing } from 'ts/pages/landing/landing';
+
+import 'basscss/css/basscss.css';
+import 'less/all.less';
```

Comment:
Remove both.

Reply:
Why? Wouldn't they write CSS in that file?

Reply:
No. We don't want any CSS or LESS files anymore. All styling should be in the components themselves.